### ISSUE TITLE
Consider Supporting a registry_uri Property

### ISSUE THREAD

**Issue by theory** [theory/sqitch#241]
Some folks want to be able to specify a different user, schema, database, or even host (#238) for the registry of changes. So consider adding a target attribute to allow it to be set via the configuration file.

**Comment by guicampos** [comment]
Hello!
Any update on this feature?

Thanks!

**Comment by theory** [comment]
No, no news, sorry. I've not been able to give Sqitch much attention for a while, and neither I nor my employer has a pressing need for this feature. Do you think it's something you'd like to work on?

**Comment by guicampos** [comment]
I'm sorry, I don't think I have the skills to implement this. I'll try to take a look, but I think I don't.
